docs/docs/Components/components-tools.md

```diff
@@ -48,7 +48,52 @@ This component searches and retrieves papers from [arXiv.org](https://arXiv.org)
 
 ## Astra DB Tool
 
-The `Astra DB Tool` allows agents to connect to and query data from Astra DB collections.
+This component allows agents to query data from Astra DB collections.
+
+To use this tool in a flow, connect it to an **Agent** component.
+The flow looks like this:
+
+![Astra DB JSON tool connected to an Agent](/img/starter-flow-simple-agent.png)
+
+The **Tool Name** and **Tool Description** fields are required for the Agent to decide when to use the tool.
+**Tool Name** cannot contain spaces.
+
+The values for **Collection Name**, **Astra DB Application Token**, and **Astra DB API Endpoint** are found in your Astra DB deployment. For more information, see the [DataStax documentation](https://docs.datastax.com/en/astra-db-serverless/databases/create-database.html).
+
+In this example, an **OpenAI** embeddings component is connected to use the Astra DB tool component's **Semantic Search** capability.
+To use **Semantic Search**, you must have an embedding model or Astra DB Vectorize enabled.
+If you try to run the flow without an embedding model, you will get an error.
+
+Open the **Playground** and ask a question about your data.
+The Agent uses the **Astra DB Tool** to return information about your collection.
+
+### Astra DB tool parameters
```

Comment:
Minor NIT: should "tool" be capitalized here like it is at the beginning?

Reply:
Heh, other way around, it shouldn't be capitalized in the beginning. Fixed